.github/workflows/tests.yml

```diff
@@ -40,6 +40,8 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           fail_ci_if_error: true
```

Comment:
Do you think we should keep this to `true` @sanders41 and @brunoocasali?

Reply:
Having it set to true will fail CI if a PR reduces the test coverage therefore making sure any new code also has tests to go with it. If you all would rather set it to false, allowing new code without tests, we can.

Reply:
Nope, I go with your decision here 😊 
Let's keep it! thank you for the explanation